components/brave_rewards/browser/contribution_report_info.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
this needs to be removed, probably bad rebase

Reply:
Fixed.